sphinx/builders/linkcheck.py

```diff
@@ -56,6 +56,11 @@ def redirect_request(self, req, fp, code, msg, headers, newurl):
 
 class HeadRequest(Request):
     """Subclass of urllib2.Request that sends a HEAD request."""
+    def __init__(self, *args, **kwargs):
+        super(HeadRequest, self).__init__(*args, **kwargs)
```

Comment:
In py2, this line causes following error because the `Request` is old-style class. 
```
(line   16) broken    https://tcpdf.org/ - super() argument 1 must be type, not classobj
```
Please use `Request.__init__(*args, **kwargs)` instead.